patches/build-svg.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/BUILD.gn b/BUILD.gn
+index 057b8b0d89..8808059a93 100644
+--- a/BUILD.gn
++++ b/BUILD.gn
+@@ -22,6 +22,7 @@ declare_args() {
+   skia_use_angle = false
+   skia_use_egl = false
+   skia_use_expat = true
++  esy_skia_enable_svg = false
+   skia_use_fontconfig = is_linux
+   skia_use_freetype = is_android || is_fuchsia || is_linux
+   skia_use_icu = !is_fuchsia && !is_ios && !is_win  # TODO: Windows
+@@ -164,6 +165,10 @@ config("skia_private") {
+     include_dirs += [ "$skia_llvm_path/include" ]
+   }
+ 
```

Comment:
Was curious what these were for, but you answered my question on Discord - you took the time to split these out so that we could re-apply them when upgrading Skia. Thanks for thinking about creating a smooth upgrade path!